mk-files/docs.mk

```
@@ -84,20 +86,23 @@ release-docs-staging:
 
 	git clone git@github.com:confluentinc/docs-confluent-cli.git $(DOCS_CONFLUENT_CLI) && \
 	cd $(DOCS_CONFLUENT_CLI) && \
-	git checkout $(STAGING_BRANCH) && \
-	$(SED) -i 's/export RELEASE_VERSION=.*/export RELEASE_VERSION=$(NEXT_PATCH_VERSION)-SNAPSHOT/g' settings.sh && \
+	git fetch && \
+	git checkout -b update-settings-$(CURRENT_SHORT_MINOR_VERSION)-post origin/$(CURRENT_SHORT_MINOR_VERSION)-post && \
```

Is this environment variable prefixed with a "v", let's add one to the branch name if not
```suggestion
	git checkout -b release-docs-staging-$(CURRENT_SHORT_MINOR_VERSION) origin/$(CURRENT_SHORT_MINOR_VERSION)-post && \
```